src/libs/clientHelpers/createClient.ts

```diff
@@ -29,7 +28,7 @@ export const createClient = async (instance: string): Promise<CozyClient> => {
       clientKind: 'mobile',
       clientName: await getClientName(),
       shouldRequireFlagshipPermissions: true,
-      certificationConfig: { androidSafetyNetApiKey }
+      certificationConfig: {}
```

Comment:
Shouldn't we put the `cloudProjectNumber` here?

Reply:
> Shouldn't we put the `cloudProjectNumber` here?

yes, has to be the prod number. for dev/local, it doesn't matter as all certification request will fail, so I'm thinking of always using the prod number regardless of env